src/ServiceStack.EventStore/Dispatcher/EventDispatcher.cs

```diff
@@ -46,7 +46,7 @@ public EventDispatcher()
                     var request = new BasicRequest();
                     request.Headers.AddAll(headers);
 
-                    await HostContext.ServiceController.ExecuteAsync(typedEvent, request);
+                    await HostContext.ServiceController.ExecuteAsync(typedEvent, request,false);
```

Comment:
It was a mistake. I've reviewed and changed to true in the new commit. Thanks for the note :)